dotnet/src/SemanticKernel/Planning/Plan.cs

```diff
@@ -86,15 +86,18 @@ public sealed class Plan : ISKFunction
     public CompleteRequestSettings RequestSettings { get; private set; } = new();
 
     #endregion ISKFunction implementation
-
+#pragma warning disable CS8618 // Non-nullable field must contain a non-null value when exiting constructor. Consider declaring as nullable.
```

Comment:
We should implement this code in a way which will avoid `non-nullable` warning suppressions.

Reply:
Done, By changing it back to static .